arches_component_lab/views/api/widgets.py

```diff
@@ -26,18 +27,32 @@ def update_i18n_properties(response):
 
 class WidgetDataView(View):
     def get(self, request, graph_slug, node_alias):
-        card_x_node_x_widget = models.CardXNodeXWidget.objects.filter(
-            node__graph__slug=graph_slug,
-            node__alias=node_alias,
-            node__source_identifier_id__isnull=True,
-        ).first()
+        card_x_node_x_widget = (
+            models.CardXNodeXWidget.objects.filter(
+                node__graph__slug=graph_slug,
+                node__alias=node_alias,
+                node__source_identifier_id__isnull=True,
+            )
+            .select_related("node")
+            .get()
+        )
 
         response = update_i18n_properties(
             JSONDeserializer().deserialize(
                 JSONSerializer().serialize(card_x_node_x_widget)
             )
         )
 
+        datatype = DataTypeFactory().get_instance(card_x_node_x_widget.node.datatype)
+        # When dropping support for v7.6, try/except can be removed
+        try:
+            response["config"]["defaultValue"] = datatype.transform_value_from_tile(
+                response["config"].get("defaultValue", None)
+            )
+        except AttributeError:
+            # Handle the case where the datatype does not have a transform_value_from_tile method
+            pass
```

Comment:
@apeters this is what Johnathan is doing now, to avoid mixing in knowledge about the reference datatype. I was suggesting that getting this hook into v8 is preferable to a try/except here (hence, "when dropping support for v7.6....")